### PR TITLE
Add category images

### PR DESCRIPTION
**Pull request by asmega**
users can associate a small and large image to a category
via the category page

### CODE REVIEW

**Comment by Tr4pSt3R** [comment]
I run the spec and this is one of the errors I am getting: 
```
pry(#<RSpec::ExampleGroups::CategorySerializer::English>)> subject.as_json
ArgumentError: bad argument (expected URI object or URI string)
from /opt/rubies/2.1.2/lib/ruby/2.1.0/uri/common.rb:537:in `convert_to_uri'
```

spec - code:
```
it 'returns the contents of the category' do
  expect(subject.as_json).to eq(expected_en)
end
```

Could you run your specs and see if this is the case for you too please @asmega?

File: app/serializers/category_serializer.rb [hunk 1]
```diff
@@ -1,10 +1,38 @@
...
+
+  def small_image_url
+    if has_small_image?
+      URI.join(ActionController::Base.asset_host, object.small_image.try(:file).try(:url).to_s).to_s
+    end
+  end
+
+  def large_image_url
+    if has_large_image?
+      URI.join(ActionController::Base.asset_host, object.large_image.try(:file).try(:url).to_s).to_s
```
Comment (tomas-stefano) [inline app/serializers/category_serializer.rb]: Can we move this to a private method?

Comment (tomas-stefano) [inline app/serializers/category_serializer.rb]: By the way, Is Code Climate suppose to catch these changes. Is it running for Contento/CMS? cc @darokel

File: config/environments/production.rb [hunk 2]
```diff
@@ -55,7 +55,7 @@
   # config.cache_store = :mem_cache_store
 
   # Enable serving of images, stylesheets, and JavaScripts from an asset server.
-  # config.action_controller.asset_host = "http://assets.example.com"
+  config.action_controller.asset_host = "http://comfy.moneyadviceservice.org.uk"
```
Comment (tomas-stefano) [inline config/environments/production.rb]: Double quotes. Is this not better on a ENV var?

Comment (asmega) [inline config/environments/production.rb]: why would this be better as an ENV var?

Comment (tomas-stefano) [inline config/environments/production.rb]: This config is served by production, but dev/test is diff. If something that could differ from environment it is better to move to a configuration management or an ENV var You already have https://github.com/moneyadviceservice/cms/blob/master/.env.dev where you can use those variables.

**Comment by asmega** [comment]
@Tr4pSt3R my fault. now fixed

File: app/serializers/category_serializer.rb (hunk 1 above)
Comment (asmega) [inline app/serializers/category_serializer.rb]: done

File: config/environments/production.rb (hunk 2 above)
Comment (asmega) [inline config/environments/production.rb]: I've swapped the quotes and add the missing test environment config. I'm not sure how using .env.dev will work. There is no corresponding .env file on QA nor are the all the environment variables found that file present in configuration management (scripts repo)

**Comment by darokel** [comment]
Just one comment on approach but apart from that the code is :+1:

**Pull request merged**